src/backend/api/meals.js

```diff
@@ -2,13 +2,76 @@ const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
-router.get("/", async (request, response) => {
+// http://localhost:5000/api/meals	GET,	Returns all meals
+router.get("/", async (req, res) => {
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
+    const meals = await knex("meal").select("id", "title");
+
+    meals.length === 0
+    ? res.status(404).send("Meals not found")
+    : res.status(200).send(meals);
```

Comment:
I would like to suggest that all the responses from the GET endpoints should be json object rather then ==> res.status(404).send("Meals not found")